src/main/java/ru/octol1ttle/flightassistant/computers/autoflight/PitchController.java

```diff
@@ -48,6 +51,10 @@ public void tick() {
         } else {
             smoothSetPitch(targetPitch, time.deltaTime);
         }
+
+        if (chunkStatus.shouldCorrectTerrain()){
+            smoothSetPitch(FAMathHelper.toDegrees(MathHelper.atan2(chunkStatus.recoverVector.y, chunkStatus.recoverVector.x)), time.deltaTime);
+        }
```

Comment:
This instance of automatic pitch correction isn't mutually exclusive with autopilot pitch movement, which will lead to these systems fighting